$NetBSD$

Inhibit a couple of "phone home" options that are enabled by default.  These
should be opt-in rather than opt-out.

The ConfigData.pm module determines what the default setting is for new
options during an upgrade as well as on an initial installation.

--- scripts/ZoneMinder/lib/ZoneMinder/ConfigData.pm.in.orig	2018-12-08 14:22:36.000000000 +0000
+++ scripts/ZoneMinder/lib/ZoneMinder/ConfigData.pm.in
@@ -2693,7 +2693,7 @@ our @options = (
   },
   {
     name        => 'ZM_CHECK_FOR_UPDATES',
-    default     => 'yes',
+    default     => 'no',
     description => 'Check with zoneminder.com for updated versions',
     help        => q`
       From ZoneMinder version 1.17.0 onwards new versions are
